### PR TITLE
Feature/cplp 1025 Error component

### PR DESCRIPTION
**Pull request by PI-FerasAhmad**
- ErrorPage Component in storybook added
- ErrorBoundary in portal added
- ErrorBonday Component to router added
- Error property in redux added to use and open error page

### CODE REVIEW

File: cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          sx={{
+            fontFamily: 'LibreFranklin-Light',
+          }}
+          variant="body1"
+        >
+          {title}
+        </Typography>
+      )}
+
+      {title && (
```
Comment (floriansommerraysono) [inline cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorDescription.tsx:41]: What was the idea to have a dependency with title for description. My thought on an error page is if I have any information I should show it.

File: cx-portal/src/services/AccessService.tsx [hunk 2]
```diff
@@ -189,6 +188,7 @@ const ALL_PAGES: IPage[] = [
     element: <Translator />,
   },
   { name: PAGES.HELP, element: <Help /> },
+  // { name: PAGES.ERROR, element: <ErrorBondary /> },
```
Comment (floriansommerraysono) [inline cx-portal/src/services/AccessService.tsx:191]: Can we remove this?

**Comment by jjeroch** [comment]
++ @lavanya-bmw  & @nidhigarg-bmw  for your information. This is not usable in the registration app, however in future a good element for the portal development

**Review by floriansommerraysono — APPROVED** [review]

File: cx-portal/src/services/AccessService.tsx (hunk 2 above)
Comment (PI-FerasAhmad) [reply cx-portal/src/services/AccessService.tsx:191]: yes we can remoe it, thanks ;-)

File: cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorDescription.tsx (hunk 1 above)
Comment (PI-FerasAhmad) [reply cx-portal-shared-components/src/components/basic/ErrorPage/components/ErrorDescription.tsx:41]: the error descriptions component have a title and descriptions. so I will add both: { title && description && (

**Pull request merged**